examples/esp32/esp32-server.rs

```diff
@@ -62,18 +91,35 @@ fn main() -> anyhow::Result<()> {
     #[cfg(not(feature = "qemu"))]
     let robot = None;
 
+
     #[cfg(feature = "qemu")]
-    let (ip, _eth) = {
+    let (ip, _blocking_eth) = {
         info!("creating eth object");
-        let eth = eth_configure(
-            &sys_loop_stack,
-            Box::new(esp_idf_svc::eth::EspEth::wrap(EthDriver::new_openeth(
-                periph.mac,
-                sys_loop_stack.clone(),
-            )?)?),
+
+        let ip =  Ipv4Addr::new(10, 1, 12, 187);
+        let ip_configuration = embedded_svc::ipv4::Configuration::Client(ClientConfiguration::Fixed(ClientSettings{
```

Comment:
what does this code do?

Reply:
this sets the ethernet's ip [configuration](https://docs.rs/embedded-svc/0.25.1/embedded_svc/ipv4/enum.Configuration.html) to be a client, with a [fixed](https://docs.rs/embedded-svc/0.25.1/embedded_svc/ipv4/enum.ClientConfiguration.html) ip instead of using dhcp. The original implementation had this IP address so I used it to keep close to the original code.

Reply:
I see, we don't need this the IP was just the qemu-host ip. This way we can connect through emu with http2 & wwebRTC